scripts/linters/pylint_extensions_test.py

```diff
@@ -3257,3 +3257,93 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class ConcatenationCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(ConcatenationCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.StringConcatenationChecker)
+        self.checker_test_object.setup_method()
+
+    def test_string_concatenation_with_string_operands_raises_error(self):
+        binop_node1, binop_node2, binop_node3 = astroid.extract_node(
+            """
+        'string1' + 'string2' #@
+        a + 'string1' #@
+        'string2' + b #@
+        """)
```

Comment:
Move right by 4 spaces to correct the indentation as it looks a bit weird currently, i.e keep all the lines indented with 4 space shift from line where `(` starts. (If the suggested change raises lint error please let me know)
```suggestion
            """
            'string1' + 'string2' #@
            a + 'string1' #@
            'string2' + b #@
            """)
```

Reply:
Yes, you are right !! Done